CONTRIBUTING.md

```diff
@@ -140,4 +140,4 @@ The project is still in its early stages and under active development, so it pos
 
 | Package      | Link                                                                 |
 | ------------ | -------------------------------------------------------------------- |
-| @rspack/core | [1.0.1](https://github.com/web-infra-dev/rspack/releases/tag/v1.0.1) |
+| @rspack/core | [1.0.3](https://github.com/web-infra-dev/rspack/releases/tag/v1.0.3) |
```

Comment:
can we just link to the @rslib/core package.json

Reply:
@rslib/core package.json only contain @rsbuild/core, we usually do rspack overrides. Maybe in the future, we need a table to show the version correspondence which automatically generated